ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -34,6 +34,8 @@
 
   public static final String KSQL_TIMESTAMP_COLUMN_INDEX = "ksql.timestamp.column.index";
 
+  public static final String STRING_TIMESTAMP_FORMAT = "ksq.timestamp.string.format";
```

Comment:
Shouldn't this be `ksql.` to match the other keys?

Reply:
Good spot @jthompson6  Do you want to submit a PR to fix?